src/cli/info.rs

```diff
@@ -68,22 +87,94 @@ impl Display for Info {
                 pi.manifest_path.to_string_lossy()
             )?;
 
-            writeln!(f, "Tasks:")?;
-            for c in &pi.tasks {
-                writeln!(f, "  - {}", c)?;
+            if let Some(count) = pi.package_count {
+                writeln!(f, "{:20}: {}", "Dependency count", count)?;
+            }
+
+            if let Some(size) = &pi.cache_size {
+                writeln!(f, "{:20}: {}", "Cache size", size)?;
+            }
+
+            if let Some(update_time) = &pi.last_updated {
+                writeln!(f, "{:20}: {}", "Last updated", update_time)?;
+            }
+
+            if pi.platforms.len() > 0 {
+                for (i, p) in pi.platforms.iter().enumerate() {
+                    if i == 0 {
+                        writeln!(f, "{:20}: {}", "Target platforms", p)?;
+                    } else {
+                        writeln!(f, "{:20}: {}", "", p)?;
+                    }
+                }
+            }
+
+            if pi.tasks.len() > 0 {
+                for (i, t) in pi.tasks.iter().enumerate() {
+                    if i == 0 {
+                        writeln!(f, "{:20}: {}", "Tasks", t)?;
+                    } else {
+                        writeln!(f, "{:20}: {}", "", t)?;
+                    }
+                }
             }
         }
 
         Ok(())
     }
 }
 
+// Returns the size of a directory
```

Comment:
These doc comments should be tripple slashes.

Reply:
Sure.